samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,9 +4,9 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-&mram1x {
-	/delete-node/ cpuapp_rw_partitions;
+/delete-node/ &cpuapp_rw_partitions;
```

Comment:
Is there a reason to move it outside? Previous method did not work?

Reply:
Yes, it didn't work - the cpuapp_rw_partitions node was not deleted. We didn't get an error, because we weren't moving storage and dfu partition defined in default memory map, but now when I'm moving the storage partition i was getting an error that it's defined in two different addresses.